proto/greenfield/storage/events.proto

```diff
@@ -627,3 +627,18 @@ message EventCancelUpdateObjectContent {
     (gogoproto.nullable) = false
   ];
 }
+
+message EventSetBucketFlowRateLimit {
```

Comment:
Does this need any change on greenfieldscan, please talk to downstream service dev.